frontend/src/app/models/folder/folder.ts

```diff
@@ -0,0 +1,12 @@
+import { UserDto } from '../user/user-dto';
+
+export interface Folder{
```

Comment:
We need to decide on naming. Either these models should be common entities or Dtos